src/components/ec-checkbox/ec-checkbox.vue

```diff
@@ -129,6 +139,22 @@ export default {
       },
     },
   },
+  watch: {
+    indeterminate(newValue) {
+      this.updateIndeterminate(newValue);
+    },
+  },
+  mounted() {
+    this.updateIndeterminate(this.indeterminate);
```

Comment:
This is being called here in order to check if we have to mark as indeterminate the checkbox as soon as the component is mounted, right? Then maybe we could try to use the `immediate` option in the watch function instead:
```
watch: {
  indeterminate: {
    immediate: true,
    handler(newValue) {
      this.updateIndeterminate(newValue);
    },
  },
},
```

Reply:
`updateIndeterminate` does use `$refs` which are only available after component has been mounted. `immediate` watchers are triggered before mounted, so the input's ref is still null and it won't get set.